app/views/students/show.html.erb

```diff
@@ -1,34 +1,83 @@
-      <%= link_to 'Edit Profile', edit_user_registration_path, class: "btn btn-outline-dark" %>
-    </div>
-  </div>
-  <div>
-    <h6><%= pluralize @student.groups.size, 'Class' %></h6>
-    <% @student.groups.each do |group| %>
-      <p>
-        <%= group.class_name %>
-        <%= link_to student_students_group_path(@student, @students_groups.find_by(group: group)),
-            method: :delete,
-            data: { confirm: "Are you sure you want to leave this class?" } do %>
-            <i class="fas fa-sign-out-alt"></i>
+
+
+
+<div class="container mt-4">
+      <h3 style="text-align: center; background: white; box-shadow: 0 0 15px rgba(0,0,0,0.2);  padding: 16px;"> Welcome to Your Student Profile </h3>
+
+  <div class="row mt-5">
+    <div class="col-12 col-lg-4">
+      <div class="d-flex" style="align-items: center;">
+        <div style="width: 150px; margin-right: 20px;">
+
+          <% if current_user.profile_photo? %>
+            <%= cl_image_tag current_user.photo.key, height: 150, width: 150, crop: :thumb, gravity: :face, class: "avatar-xl" %>
+          <% else %>
+            <i class="fas fa-user-circle" style="font-size: 150px;"></i>
         <% end %>
-      </p>
+        </div>
+        <div style="max-width: 250px;">
+          <h5><%= @student.first_name %> <%= @student.last_name %></h5>
+          <p><%= current_user.email %></p>
+          <%= link_to 'Edit Profile', edit_user_registration_path, class: "btn btn-outline-dark" %>
+        </div>
+      </div>
+      <br>
+
+
+
+      <h4><%= pluralize @student.groups.size, 'Class' %></h4>
+        <% @student.groups.each do |group| %>
+        <div class="card-product" style="margin-top: 0.7em">
+          <%= image_tag "classroom.svg" %>
+            <div class="card-product-infos">
+              <h2><%= group.class_name %></h2>
+              <p> Leave the class <%= link_to student_students_group_path(@student, @students_groups.find_by(group: group)),
+              method: :delete,
+              data: { confirm: "Are you sure you want to leave this class?" } do %>
+              <i class="fas fa-sign-out-alt"></i> </p>
+            </div>
+        </div>
+      <% end %>
     <% end %>
+
     <br>
-    <%= simple_form_for [@student, @students_group] do |f| %>
-      <%= f.association :group, input_html: { class: "beautiful-dropdown" }, include_hidden: false, label: 'Choose a Class' %>
-      <%= f.submit "Join Class", class: "btn btn-primary" %>
-    <% end %>
+      <%= simple_form_for [@student, @students_group] do |f| %>
+        <%= f.association :group, input_html: { class: "beautiful-dropdown" }, include_hidden: false, label: 'Choose a Class' %>
+        <%= f.submit "Join Class", class: "btn btn-primary" %>
+      <% end %>
+    </div>
+
+    <div class="col-12 col-lg-8">
+    <h4> Homework Manager </h4>
+
+      <div class="notification">
+            <%= image_tag "question.svg", class: "avatar-large" %>
+        <div class="notification-content">
```

Comment:
what exactly will appear here, instead of the notifications placeholders? I think it makes more sense to have the homeworks appearing in a separate page. the notifications would look cool in the dashboard. I don't see people going to their profile a lot, it's usually  a place you go quite rarely

Reply:
the dashboard will be only for teacher the student has only his profile page and can get notifications there that redirect them to actual homeworks. We can put something else here but tbh I have no other good ideas

Reply:
So this will be very easy to adjust as now it's pure html and no ruby, once the whole backend is working you can put in this notifications whatever you want